### PR TITLE
Clean up more code for live_elo page

### PR DESCRIPTION
**Pull request by ppigazzini**
Companion PR for https://github.com/glinscott/fishtest/pull/1397

Thanks to @vdbergh for the idea and @dav1312 for the review.

### CODE REVIEW

**Comment by ppigazzini** [comment]
DEV updated, LGTM, please @dav1312 have a look at your convenient time.

File: server/fishtest/static/css/live_elo.css [hunk 1]
```diff
@@ -1,30 +0,0 @@
...
-  color: #5f9960;
-}
-
-a:focus {
-  color: #497449;
-}
-
-td:last-child {
-  word-break: break-word;
-}
```
Comment (dav1312) [inline server/fishtest/static/css/live_elo.css:30]: Looks like removing this is adding a very small horizontal scroll on my phone

File: server/fishtest/static/css/live_elo.css [hunk 2]
```diff
@@ -1,30 +0,0 @@
-div.error {
-  display: none;
-}
```
Comment (dav1312) [inline server/fishtest/static/css/live_elo.css:3]: We could also remove all `error` related code right?

https://github.com/glinscott/fishtest/blob/2dc3970bf45ad5c3415e2e658cffed9e1d7b4d3c/server/fishtest/templates/tests_live_elo.mak#L25

https://github.com/glinscott/fishtest/blob/2dc3970bf45ad5c3415e2e658cffed9e1d7b4d3c/server/fishtest/static/js/live_elo.js#L146

Comment (vdbergh) [reply server/fishtest/static/css/live_elo.css:3]: I think so.

File: server/fishtest/static/css/live_elo.css (hunk 1 above)
Comment (vdbergh) [reply server/fishtest/static/css/live_elo.css:30]: I find horizontal scrolling rather annoying. Perhaps we could move this to application.css?

File: server/fishtest/static/css/live_elo.css (hunk 2 above)
Comment (ppigazzini) [reply server/fishtest/static/css/live_elo.css:3]: Correct, the alert_() function was dropped in #1397
DEV updated.

File: server/fishtest/static/css/live_elo.css (hunk 1 above)
Comment (ppigazzini) [reply server/fishtest/static/css/live_elo.css:30]: Confirmed. Embarrassedly I haven't idea how to fix...

Comment (ppigazzini) [reply server/fishtest/static/css/live_elo.css:30]: I added a style tag in the mako template to break word for the last child, it works.

**Pull request merged**